### PR TITLE
Add LoginView

### PR DESCRIPTION
**Pull request by brevansio**
Adds a Keychain enabled login view.　Its not quite what I imagined thanks to some SwiftUI limitations, but the popover doesn't look too bad.

loginComplete

### CODE REVIEW

File: PRChecker/Shared/ContentView.swift [hunk 1]
```diff
@@ -8,9 +8,43 @@
...
+                    Button {
+                        // TODO: Settings Page
+                    } label: {
+                        Text(.init(systemName: "gearshape"))
+                            .font(.system(size: 45))
+                            .foregroundColor(.secondary)
+                    }
+                        .buttonStyle(.link)
+                }
+                .frame(alignment: .leading)
```
Comment (Ckitakishi) [inline PRChecker/Shared/ContentView.swift]: Ah, I haven't understood the difference between `VStack().frame(alignment: .leading)` and `VStack(alignment: .leading)`..

**Review by Ckitakishi — APPROVED** [review]
LGTM!

File: PRChecker/Shared/ContentView.swift (hunk 1 above)
Comment (brevansio) [reply PRChecker/Shared/ContentView.swift]: Oh... neither do I lol... Let me try it out

Comment (brevansio) [reply PRChecker/Shared/ContentView.swift]: Ah, `VStack(alignment: .leading)` sets the internal alignment to `.leading` and `.frame(alignment: .leading)` sets the alignment of the VStack itself.

...either way when I tested it out deleting both didn't change the UI so I'll simplify it by removing it.

Comment (Ckitakishi) [reply PRChecker/Shared/ContentView.swift]: > ...either way when I tested it out deleting both didn't change the UI so I'll simplify it by removing it.

Unexpected ending😂

**Pull request merged**